server/__tests__/meilisearch.test.js

```diff
@@ -145,13 +145,15 @@ describe('Tests content types', () => {
           },
         },
         config: { get: jest.fn(() => ({ restaurant: jest.fn() })) },
+        log: mockLogger,
       },
     })
 
     await service.addContentTypeInMeiliSearch({ contentType: 'restaurant' })
 
-    expect(Meilisearch().index).toHaveBeenCalledWith('restaurant')
-    expect(Meilisearch().index().addDocuments).toHaveBeenNthCalledWith(
+    const client = new Meilisearch({ host: '' })
+    expect(client.index).toHaveBeenCalledWith('restaurant')
+    expect(client.index('restaurant').addDocuments).toHaveBeenNthCalledWith(
```

Comment:
Why did you have to change these lines?

Reply:
My typescript plugin on my IDE was not happy about it 😅